src/app/utils/client.ts

```diff
@@ -0,0 +1,7 @@
+import { createPublicClient, http } from "viem";
+import { skaleNebulaTestnet } from "viem/chains";
+
+export const publicClient = createPublicClient({
```

Comment:
Public client can be created in the providers file itself, no need to put it in a separate file.